websites/S/Sushi Scan/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.12",
+	"apiVersion": 1,
+	"author": {
+		"id": "723471302123323434",
+		"name": "redstom"
+	},
+	"service": "Sushi Scan",
+	"description": {
+		"en": "Read comics on Sushi Scan.",
+		"fr": "Lisez des mangas et bandes dessinées sur Sushi Scan."
+	},
+	"url": [
+		"sushiscan.net",
+		"www.sushiscan.net"
+	],
+	"version": "1.0.0",
+	"logo": "https://sushiscan.net/wp-content/uploads/IconSushiScan.png",
+	"thumbnail": "https://sushiscan.net/wp-content/uploads/IconSushiScan.png",
```

Comment:
The thumbnail should preferably be a [wide promotional card](https://i.imgur.com/3QfIc5v.jpg) or a [screenshot](https://i.imgur.com/OAcBmwW.png) if the first is not available.